### PR TITLE
Add bike goal DACC service

### PR DESCRIPTION
**Pull request by paultranvan**
The objective is to compute the number of days including a commute trip
in bicycle, and send it to the DACC, daily.
This way, users will be able to compare themselves to others users.

### CODE REVIEW

File: manifest.webapp [hunk 1]
```diff
@@ -100,7 +100,12 @@
     "daccMonthlyCO2": {
       "type": "node",
       "file": "services/daccMonthlyCO2/coachco2.js",
-      "trigger": "@every 720h"
+      "trigger": "@monthly on the 1"
```
Comment (Crash--) [inline manifest.webapp:103]: wanted this change? If yes, should it be in another commit?

File: src/hooks/useFetchDACCAggregates.jsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { useState, useEffect } from 'react'
-import { fetchMonthlyAverageCO2FromDACCFor11Month } from 'src/lib/dacc'
+import { fetchMonthlyAverageCO2FromDACCFor11Month } from 'src/lib/daccMonthlyCO2'
```
Comment (Crash--) [inline src/hooks/useFetchDACCAggregates.jsx:2]: should be in the previous commit?

File: src/hooks/useFetchDACCAggregates.spec.jsx [hunk 3]
```diff
@@ -1,12 +1,12 @@
 import { renderHook } from '@testing-library/react-hooks'
 import React from 'react'
 import useFetchDACCAggregates from 'src/hooks/useFetchDACCAggregates'
-import { fetchMonthlyAverageCO2FromDACCFor11Month } from 'src/lib/dacc'
+import { fetchMonthlyAverageCO2FromDACCFor11Month } from 'src/lib/daccMonthlyCO2'
```
Comment (Crash--) [inline src/hooks/useFetchDACCAggregates.spec.jsx:4]: same?

File: src/lib/dacc.js [hunk 4]
```diff
@@ -64,5 +120,28 @@ export const runDACCService = async client => {
...
     )
   }
+
   return shouldRestartService
 }
+
+/**
+ * Run the bike goal DACC service
+ *
+ * @param {import("cozy-client/dist/index").CozyClient}} client - The cozy-client instance
```
Comment (Crash--) [inline src/lib/dacc.js:130]: tips: You can create a typedef: 

```
/**
 * @typedef {import("cozy-client/dist/index").CozyClient} CozyClient
 */
```

and then juste 
```
@param {CozyClient} client
```

so much easiiiier :D

File: src/lib/daccBikeGoal.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+import { countUniqDays } from './timeseries'
+
+/**
+ * The group name provided by flag
+ *
+ * @param {import("cozy-client/dist/index").CozyClient}} client - The cozy-client instance
+ * @returns {string} The group name
+ */
+export const getBikeGroupName = async client => {
+  await flag.initialize(client)
```
Comment (Crash--) [inline src/lib/daccBikeGoal.js:20]: note to myself: we should provide a `runService` command that handle the flag init for us.

File: src/targets/services/daccMonthlyCO2.js [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import fetch from 'node-fetch'
-import { DACC_SERVICE_NAME } from 'src/constants'
+import { DACC_MONTHLY_CO2_SERVICE_NAME } from 'src/constants'
 import schema from 'src/doctypes'
-import { runDACCService } from 'src/lib/dacc'
+import { runMonthlyCO2DACCService } from 'src/lib/dacc'
```
Comment (Crash--) [inline src/targets/services/daccMonthlyCO2.js:4]: I think it should be part of the previous commit

File: manifest.webapp (hunk 1 above)
Comment (paultranvan) [reply manifest.webapp:103]: Yes it is wanted, and yes it should be. It is now :) 
ff768380443ff5d9a40f73def3e8b0fa7f2fec72

File: src/lib/dacc.js (hunk 4 above)
Comment (paultranvan) [reply src/lib/dacc.js]: I don't know about easier, but more elegant for sure :)

Comment (Crash--) [reply src/lib/dacc.js]: Easier since you don't have to look for the import and copy / paste it everywhere ^^.

File: src/lib/daccBikeGoal.js (hunk 5 above)
Comment (paultranvan) [reply src/lib/daccBikeGoal.js:25]: Where would be this command?

File: src/lib/types.js [hunk 7]
```diff
@@ -1,3 +1,25 @@
...
+ */
+
+/**
+ * Account's doctype
+ * 
+ * See https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.accounts.md
+ *
+ * @typedef {object} Account
+ * @property {string} _id - The account's id
+ *
```
[the remark this answers is not in the document]
Comment (paultranvan) [reply src/lib/types.js]: I didn't think to check that :face_exhaling:

File: src/lib/dacc.js (hunk 4 above)
Comment (paultranvan) [reply src/lib/dacc.js]: Ah, for me the IDE kindly auto-completes
